docs/standard/serialization/snippets/system-text-json-how-to/csharp/DeserializeRequiredPropertyAttrReg.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace SystemTextJsonSamples
+{
+    public class DeserializeRequiredPropertyAttrReg
```

Comment:
This might be long, but let's spell all the words in this name in full, here and in other types/methods below.